Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM centos:7
+
+EXPOSE 10000 10080 10443
+
+COPY . /libjaypea
+
+RUN ["/libjaypea/bin/setup-centos7.sh"]
+
+RUN groupadd -r -g 1000 user && useradd -r -m -g user -u 1000 user
```

Comment:
https://docs.docker.com/engine/reference/builder/ basically. Run will run the specified command, entrypoint will do some other janky stuff